src/mame/pc/sis630.cpp

```diff
@@ -400,8 +400,24 @@ ROM_START(zidav630e)
 //  ROMX_LOAD( "V630e104.bin",     0x040000, 0x040000, CRC(?) SHA1(?), ROM_BIOS(1) )
 ROM_END
 
-/*
- * Arcade based GameCristal
+
+/* GameCristal is a PC-based multigame arcade (with an unknown emulator).
+   It uses a small external PCB from Azkoyen with an MCU (unknown type) for the JAMMA interface:
+                              _______________         _______
+   ____--__-- _______________|   DB-25      |________| DB-9 |_____
+  |   |__||__| <-Jacks       |______________|        |______|    |
+  | _________                                               ___  |
+  ||o o o o |<-Power                     ________________  |  |  |
+  |                                     | MCU           |  |  |  |
+  |                                     |_______________|  |__|  |
+  |                                      _________   Xtal        |
+  |                                     |DIPS x 8|   5 MHz       |
+  |                                                              |
+  |    Test sw->(o)         _________   _________   _________    |
+  | Service sw->(o)         SN74HC245N  SN74HC245N  SN74HC245N   |
+  |_________                                          ___________|
+           |_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|_|
+                          JAMMA
```

Comment:
Should be at the bottom of the specific header section, not here.

Reply:
Fixed!